core/UserManager/UserHelper.py

```diff
@@ -6,3 +6,16 @@ class UserHelper:
     def get_user_by_email(email):
         # Get a user object by exact email and return it.
         return User.objects.get(email__exact=email)
+
+    @staticmethod
+    def get_user_by_id(id):
+        return User.objects.get(id=id)
+
+    @staticmethod
+    def validate_password(password):
+        if len(password) <= 10:
+            raise serializers.ValidationError('Password must be greater than 10 characters')
```

Comment:
Getting erros on both `serializers.ValidationError`. The library and the ValidateError class were not importedo.

Also, as I mentioned on previous comment, Django has a built-in password validation, it should be worth taking a look at it.